src/Component/Style/Style.tsx

```diff
@@ -157,7 +129,7 @@ class Style extends React.Component<StyleProps, StyleState> {
     const randomId = Math.floor(Math.random() * 10000);
     const newRule: GsRule = {
       name: 'rule_' + randomId,
-      symbolizer: this.getSymbolizerFromStyleType(style)
+      symbolizer: [this.getDefaultSymbolizer(style.rules[0].symbolizer[0])]
```

Comment:
Is `style.rules[0].symbolizer[0]` available always?

Reply:
Not necessarily. Thanks for pointing out. Now using lodash's `get` function at this point and add handling if parameter in `getDefaultSymbolizer()` is undefined. See [this commit](https://github.com/terrestris/geostyler/pull/337/commits/efe89067b90763238668783c16f0f52e81b1aaed).